www/artists/afyfe/rfiveproject1.html

```diff
@@ -29,33 +29,35 @@
                 <h2 class="c1">
                     Regina 5 Sinks
                 </h2>
-                <p></p><img src="images/sink1.jpg" width="200" height="175" alt="Sink 1" style='border-width:1px;'>
+                <img class = "r5img" src="images/sink1.JPG" width="200" height="175" alt="Sink 1">
                 <br>
-                <a href="http://saskartists.ca/rbloore/work.html"><strong style='text-align:left;'>Ronald Bloor Sink</strong></a>
+                <a class = "r5" href="http://saskartists.ca/rbloore/work.html">(White on white) Ronald Bloor Sink</a>
                 <br>
                 <br>
-                <img src="images/sink3.jpg" width="200" height="175" alt="Sink 2" style='border-width:1px;'>
+                <img class = "r5img" src="images/sink2a.JPG" width="200" height="175" alt="Sink 2">
                 <br>
-                <a href="http://saskartists.ca/amckay/info.html"><strong style='text-align:left;'>Art McKay Sink</strong></a>
+                <a class = "r5" href="http://saskartists.ca/amckay/info.html">(Mandella) Art McKay Sink</a>
                 <br>
                 <br>
-                <img src="images/sink2a.jpg" width="200" height="175" alt="Sink 3" style='border-width:1px;'>
+                <img class = "r5img" src="images/sink3.JPG" width="200" height="175" alt="Sink 3">
                 <br>
-                <a href="http://saskartists.ca/tgodwin/"><strong style='text-align:left;'>Ted Godwin Sink</strong></a>
+                <a class = "r5" href="http://saskartists.ca/tgodwin/">(Tartan) Ted Godwin Sink</a>
                 <br>
                 <br>
                 <br>
                 <br>
-                <img src="images/sink4.jpg" width="200" height="175" alt="Sink 4" style='border-width:1px;'>
+                <img class = "r5img" src="images/sink4.JPG" width="200" height="175" alt="Sink 4">
                 <br>
-                <a href="http://saskartists.ca/klochhead/"><strong style='text-align:left;'>Ken Lockhead Sink</strong></a>
+                <a class = "r5" href="http://saskartists.ca/klochhead/">(Dreamscape) Ken Lockhead Sink</a>
                 <br>
                 <br>
-                <img src="images/sink5.jpg" width="200" height="175" alt="Sink 5" style='border-width:1px;'>
-                <br>
-                <a href="http://saskartists.ca/dmorton/"><strong style='text-align:left;'>Doug Morton Sink</strong></a>
+                <img class = "r5img" src="images/sink5.JPG" width="200" height="175" alt="Sink 5" >
                 <br>
+                <a class = "r5" href="http://saskartists.ca/dmorton/">(Colour Purity) Doug Morton Sink</a>
                 <br>
+                <p>
+                    "Everything I tried the way they did it, it went down the drain," said Fyfe.
```

Comment:
Same thing as I've mentioned twice now - reformat it for a proper website quote, and add proper attribution.